packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: The Imperative framework adds the value in the `set-cookie` response header to the `_availableCreds` and to the root of the `ISession` object. [#2605](https://github.com/zowe/zowe-cli/pull/2605)
```

Comment:
can we say what `_availableCreds` is?: "... to the `_availableCreds` **property/class/cat** and to the root of the `ISession` object."